<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasySiteCreator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Добавить страницу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Сохранить текущую страницу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>Путь на сайте</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Имя файла</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="run_server">
        <property name="text">
         <string>Запустить сервер</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_htmlcode">
          <property name="text">
           <string>Страница</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_database">
          <property name="text">
           <string>База данных</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Html">
          <property name="text">
           <string>Html</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Server">
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="db_button_layout">
        <item>
         <widget class="QPushButton" name="create_table">
          <property name="text">
           <string>Create table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_table">
          <property name="text">
           <string>Delete table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="insert_row">
          <property name="text">
           <string>Insert row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_row">
          <property name="text">
           <string>Remowe row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_row">
          <property name="text">
           <string>Update row</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QTextEdit" name="serveredit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Параметры сервера</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
